test/snippet/nanocourse/solution4.cpp

```diff
@@ -74,11 +72,14 @@ int main()
 
     B.construct(64, 1600);
 
-    for (auto a : B.blocks)
+    std::cout << "Superblocks:" << std::endl;
```

Comment:
Actually I think we should do `'\n'` instead of `std::endl` everywhere...

Reply:
why ?